components/brave_shields/common/brave_shield_utils.cc

```diff
@@ -47,3 +52,64 @@ ContentSetting GetBraveFPContentSettingFromRules(
 
   return CONTENT_SETTING_DEFAULT;
 }
+
+ShieldsSettingCounts GetFPSettingCountFromRules(
+    const ContentSettingsForOneType& fp_rules) {
+  ShieldsSettingCounts result = {};
+
+  for (const auto& rule : fp_rules) {
+    if (rule.primary_pattern.MatchesAllHosts()) {
+      continue;
+    }
+    if (rule.secondary_pattern.MatchesAllHosts()) {
+      if (rule.GetContentSetting() == CONTENT_SETTING_ALLOW) {
+        result.allow++;
+      } else {
+        result.aggressive++;
+      }
+    } else {
+      if (rule.GetContentSetting() == CONTENT_SETTING_BLOCK) {
+        result.standard++;
+      }
+    }
+  }
+
+  return result;
+}
+
+ShieldsSettingCounts GetAdsSettingCountFromRules(
+    const ContentSettingsForOneType& ads_rules) {
+  ShieldsSettingCounts result = {};
+
+  std::unordered_set<std::string> block_set;
```

Comment:
using `unordered_set` is discouraged in chromium, see https://chromium.googlesource.com/chromium/src/+show/master/base/containers/README.md#32

Reply:
switched to `set`